src/pages/PlanPage/subRoutes/CancelPlanPage/CancelPlanPage.test.tsx

```diff
@@ -20,15 +20,15 @@ const teamPlans = [
   {
     baseUnitPrice: 6,
     benefits: ['Up to 10 users'],
-    billingRate: 'monthly',
+    billingRate: BillingRate.MONTHLY,
     marketingName: 'Users Team',
     monthlyUploadLimit: 2500,
     value: Plans.USERS_TEAMM,
   },
   {
     baseUnitPrice: 5,
     benefits: ['Up to 10 users'],
-    billingRate: 'yearly',
```

Comment:
I noticed some billingRates say `yearly` while others say `annually` (before your changes), did you confirm the new logic accounts for this?

Reply:
yuhhhh we don't actually use yearly anywhere on the backend lol, maybe long ago we did and these mocks just never needed to be updated since we only typed it as a string vs. a const